app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,14 @@
 class CommentsController < ApplicationController
```

Comment:
:orange_circle: [OPTIONAL] It's often common practice to add /api to the URL or api/v1 with the version number in order to identify it as an API-based URL. To implement your API routes in this way you have to include your API-controllers inside a folder named api :file_folder:  (if you want to include the versions number you can create subfolders v1 :file_folder: , v2 :file_folder:  inside your api :file_folder: where each subfolder represents the version )

```
-api
    |-v1
       |-Your controllers
```
Then kindly make sure you wrap your api routes with a namespace

```
  namespace :api, defaults: { format: :json } do
    namespace :v1 do
      # your controllers
   end
end
```